DOCSP-27090 Add Atlas only stage note (#449)
mongodb/docs-compass: source/aggregation-pipeline-builder.txt

Collection` section shows 20 documents randomly sampled from the chosen
collection.

> .. note:: 

Reviewer: [suggestion]

For findability, suggest giving this note a title like "Atlas Search Stages"

```suggestion
.. note:: Atlas Search Stages
```

> .. note:: 

Reviewer: Consider placing this note in the "Create an Aggregation Pipeline" section, possibly at the end of step 1 where we instruct users to choose a stage.

>    When you connect Compass to MongoDB instances hosted on 

Reviewer: [nit]

I'm admittedly forgetting the context around this decision, but I think we want to use "deployment" rather than "instance".

 ```suggestion
   When you connect Compass to a MongoDB deployment hosted on 
```

>    When you connect Compass to MongoDB instances hosted on 

Reviewer: [suggestion]

Use the replacement `|compass-short|` rather than "Compass".

   `Atlas <https://www.mongodb.com/cloud/atlas?tck=docs_compass>`_,

>    additional Atlas only stages ``$search`` and ``$searchMeta`` are 

Reviewer: I think "Atlas only" needs a hyphen.

```suggestion
   additional Atlas-only stages ``$search`` and ``$searchMeta`` are 
```

>    additional Atlas only stages ``$search`` and ``$searchMeta`` are 

Reviewer: [suggestion]

Can we provide links to the $search and $searchMeta stages:

- [$search](https://www.mongodb.com/docs/atlas/atlas-search/query-syntax/#-search)
- [$searchMeta](https://www.mongodb.com/docs/atlas/atlas-search/query-syntax/#-searchmeta)

We can use the `:atlas:` role to simplify the syntax for these links.

Author: Sure thing, I added these as links and used the :atlas: role.

Collection` section shows 20 documents randomly sampled from the chosen
collection.

.. note:: Atlas Search Stages

   When you connect |compass-short| to a MongoDB deployment hosted on 
   `Atlas <https://www.mongodb.com/cloud/atlas?tck=docs_compass>`_,
   additional Atlas-only stages 
   :atlas:`$search </reference/atlas-search/query-syntax/#-search>` and
   :atlas:`$searchMeta </reference/atlas-search/query-syntax/#-searchmeta>`

>    are available in the Aggregation Pipeline Builder.

Reviewer: [suggestion]

Maybe add a sentence explaining when you might use these stages:

"Use these stages to perform full-text search on Atlas collections."

or something
